index.js

```diff
@@ -72,6 +72,13 @@ module.exports = {
 			rules: {
 				'@typescript-eslint/no-var-requires': off
 			}
+		},
+		{
+			files: ['*.svelte'],
+			processor: 'svelte3/svelte3',
+			rules: {
+				'@typescript-eslint/indent': off
```

Comment:
Unless something else is still enabling `@typescript-eslint/indent`, we shouldn't need to be disabling it again here.

Reply:
Yeah - I think the goal here is just to remove custom rules and leave the svelte3 linting intact + defaulted.

After this fix I'll re-run the linter on the main Svelte codebase and re-open the PR with the assert mutation :eyes: turned into a warning

Reply:
yeah, thanks for catching. that was just a sloppy copy and paste